OPERATOR.rst

```diff
@@ -267,7 +267,14 @@ create a properly tagged snapshot of the GitLab EBS volume. Run::
 
 	python scripts/create_gitlab_snapshot.py
 
-Once the snapshot is complete, edit the `GitLab Terraform`_ file, updating the
+.. FIXME: Should not have to destroy the instance to update
+		  https://github.com/DataBiosphere/azul/issues/4019
```

Comment:
There seems to be a tab in the indent here

Reply:
Fixed, Note removing the paragraph kinda wrecks the formatting, I used another code-block to keep the #FIXME above the line it refers to. If we want to use one code block the only option is to place the FIXME comment below the line used, RST cant seem to handle the comment in between the RUN:: and the code block.